src/doc/src/reference/build-scripts.md

```diff
@@ -233,7 +235,10 @@ equivalent to using [`rustc-link-lib`](#rustc-link-lib) and
 
 The `rustc-cfg` instruction tells Cargo to pass the given value to the
 [`--cfg` flag][option-cfg] to the compiler. This may be used for compile-time
-detection of features to enable [conditional compilation].
+detection of features to enable [conditional compilation]. Custom cfgs
+must either be expected using the [`cargo::rustc-check-cfg`](#rustc-check-cfg)
+instruction or usage will need to allow the `unexpected_cfgs` lint to avoid
```

Comment:
It will be great to have a link to [the lint rule](https://doc.rust-lang.org/nightly/rustc/lints/listing/warn-by-default.html#unexpected-cfgs).

Reply:
Sure, done.